app/controllers/providers/applicant_details_controller.rb

```diff
@@ -19,9 +19,12 @@ def applicant
 
     def form_params
       merge_with_model(applicant) do
-        convert_date_params('applicant')
         params.require(:applicant).permit(*Applicants::BasicDetailsForm::ATTRIBUTES)
       end
     end
+
+    def converted_params
+      convert_date_params(form_params)
+    end
```

Comment:
Just wondering why it is better to add converted params as a separate method, rather than just calling convert_date_params at the end of the form_params method?

Reply:
Fair point, it was easier to break them into separate methods while I was making the changes, but it's straightforwad to convert them as part of `form_params`.